example/ios/Podfile

```diff
@@ -105,3 +106,14 @@ def fix_deployment_target(installer)
     end
   end
 end
+
+def disable_resource_bundle_signing(installer)
+  installer.pods_project.targets.each do |target|
+    target_is_resource_bundle = target.respond_to?(:product_type) && target.product_type == 'com.apple.product-type.bundle'
+    target.build_configurations.each do |config|
+      if target_is_resource_bundle
+        config.build_settings['CODE_SIGNING_ALLOWED'] = 'NO'
```

Comment:
This is needed to workaround the required code-signing for resource bundle targets.

Initially I wanted to set the team properly but it is not trivial to get the value out of the `Developer.xcconfig` file.
This is no problem as long as we don't want to submit the example application to the App Store (it will never happen :) )

Reply:
Thanks for the comment. Might be worth add it to the code also 🙂

Reply:
That's a great idea, added comment for all workaround here: ed4dfa5